grr-ui/src/js/testSample.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
How will you be able to export the functions we're using in our application for testing?

Reply:
There are a few ways to do this, but the easiest way looks like we should break our (to be tested) functions into "bite size" pieces that are safe for exporting. We can structure our app to make this easier, or just manually break the functions ourselves. I can make that a new story to set up everything and write unit tests for our stuff to see how that works out.

Reply:
Yeah I'd like to see an example of what you want to do in terms of testing those functions before we edit the code. Shouldn't there be a way to export them already?